inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/management/DynamicWorkloadManagementPage.java

```diff
@@ -229,18 +243,19 @@ private void actionShowInfoDialog(AjaxRequestTarget aTarget, IModel<SourceDocume
         }
         try {
             return Optional.of(projectService.getProject(aProjectParam.toLong()));
-        } catch (NoResultException e) {
+        }
+        catch (NoResultException e) {
             return Optional.empty();
         }
     }
 
     public List<String> listUsersFinishedForDocument(SourceDocument aDocument)
     {
         List<String> result = new ArrayList<>();
-        for (AnnotationDocument anno: documentService.
-            listAnnotationDocuments(currentProject.getObject())) {
+        for (AnnotationDocument anno : documentService
```

Comment:
Slow - filtering should happen at SQL level in a backend service bean, e.g. documentService.